1-exercises/A-this/exercise-3.js

```diff
@@ -8,8 +8,8 @@
 
 let bankAccount = {
   currentAmount: 100,
-  deposit: function (amount) {},
-  withdraw: function (amount) {},
+  deposit: function (amount) { this.currentAmount},
+  withdraw: function (amount) {this.currentAmount},
```

Comment:
Have a think about what each of these functions should be doing. How can you use the amount parameter to achieve that? Remember you want to be changing the `currentAmount` property in the object for both of these functions